mds_loadtest/scripts/web_locust_file.py

```diff
@@ -0,0 +1,87 @@
+from locust import HttpLocust, TaskSet, seq_task, task
```

Comment:
Does locust expect a `_file` suffix in the filename? If not, that can be removed as it is redundant.

Since all we're using for load testing is locust, it is probably most idiomatic to just have `web.py` and `api.py` as the filenames.